src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('new-user');
       return;
     }
     if (isUndefinedUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('undefined-user');
       return;
     }
     if (isSecurityImage) {
+      // TODO: Newer versions of qtip will remove aria-describedby on their own when destroy() is called.
+      el.removeAttr('aria-describedby');
```

Comment:
so, if I understand it correctly, we can't really set an attribute in the image <div> element because the screenreader doesn't read it, so the workaround is to have a different <span> element with the text describing the image? Does that satisfy the accessibility standards? Asking since usually images have an alt attribute to describe them, or an aria attribute as well in the same element, not in a different one.